esphome/components/havells_solar/havells_solar.h

```diff
@@ -0,0 +1,114 @@
+#pragma once
+
+#include "esphome/core/component.h"
+#include "esphome/components/sensor/sensor.h"
+#include "esphome/components/modbus/modbus.h"
+
+namespace esphome {
+namespace havells_solar {
+
+class HAVELLSSolar : public PollingComponent, public modbus::ModbusDevice {
```

Comment:
This should be CamelCase

```suggestion
class HavellsSolar : public PollingComponent, public modbus::ModbusDevice {
```